tests/integration/cmd_dev_test.go

```diff
@@ -936,6 +936,110 @@ ComponentSettings:
 		})
 	}
 
+	for _, podman := range []bool{true, false} {
+		podman := podman
+		Context(fmt.Sprintf("multiple dev sessions with different project are running on same platform (podman=%v), same port", podman), helper.LabelPodmanIf(podman, func() {
+			const (
+				nodejsContainerPort = "3000"
+				goContainerPort     = "8080"
+				// Note: While running this test on macOS, ensure the addresses are open for request
+				// Note(contd.): Refer: https://superuser.com/questions/458875/how-do-you-get-loopback-addresses-other-than-127-0-0-1-to-work-on-os-x#458877
```

Comment:
Oh, that's weird that we cannot use any loopback address out of the box..
In this case, can you just `Skip` the test on MacOS?